events-application-rest-api/src/main/java/com/bounswe/bounswe2017group3/Controller/EventController.java

```diff
@@ -78,6 +89,18 @@ public EventController(EventRepository repository) {
         return event;
 
     }
+    
+    @RequestMapping(method=RequestMethod.DELETE, params="id", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public @ResponseBody void deleteEvent(@RequestParam("id") long id,
```

Comment:
Can you add the new visuals with this change? I'm not sure if it'd work 😅

Also, I'm so sorry, I forgot to mention that the status code of a `DELETE` request should be `204` which stands for _No Content_. Does this method respond with 204 status code?